components/generic_modal.scss

```diff
@@ -216,6 +216,32 @@
                     }
                 }
 
+                &.delete {
+                    @include primary-button;
+                    @include button-medium;
+
+                    border-color: transparent;
+                    background: $delete-color;
+                    color: $white;
+
+                    .app__body & {
+                        color: $white;
```

Comment:
would this work in other themes as well?

Reply:
Maybe we need to check https://github.com/mattermost/mattermost-webapp/blob/076d6a28ace971e8896d1bfe4e4852e0cd2c3aca/utils/utils.tsx#L404
`applytheme`function here if we change the colour of the `app_body class`. That said, I think it’s better to use `--center-channel-bg`. Although if we want it to be always White, then we should be fine.

Reply:
I don't think we should use `$white` here for the button text. This should be the `--button-color` theme variable and that should work on all themes

Reply:
Updated to use `--button-color`, styling looks consistent across themes.